app/components/Settings/Settings.react.js

```diff
@@ -42,6 +43,27 @@ class Settings extends Component {
             checkHTTPSEndpointInterval: null,
             getConnectorUrlsInterval: null
         };
+
+        const checkIfLoggedIn = () => {
+            return fetch(`/settings`, {
+                method: 'GET',
+                headers: {
+                    'Accept': 'application/json',
+                    'Content-Type': 'application/json'
+                }
+            }).then((res) => res.json().then((json) => {
+                return (res.status === 200 && json.USERS.length > 0);
+            })).catch((e) => {
+                return false;
+            }).then(isLoggedIn => {
+                this.setState({isLoggedIn});
+                if(!isLoggedIn) setTimeout(checkIfLoggedIn, 1000);
+            }).catch((e) => {
+                console.error(e);
+            });
+        };
+
+        checkIfLoggedIn();
```

Comment:
@n-riesco this wont work for electron app unfortunately.. Since electron app doesn't require log-in to access settings (or any url for that matter) .. I would suggest checking for `username`, which is stored in its parent component..

Reply:
We need to send this state: https://github.com/plotly/falcon-sql-client/blob/master/app/components/Configuration.react.js#L26
as a prop to `Settings` and take use it as a measure of user being logged-in or not. Let me know if you need more info on this.. 😸

Reply:
@tarzzz Not sure I understand, it does work on my machine.

Reply:
@chriddyp This should work for electron-app and we are letting server/headless version to handle their own certificates.. so should be good?

Reply:
@tarzzz - this logic makes sense to me for the headless version however it won't apply to the multi-user server/headless version - I'm guessing that a check for `username` will work in that case